roles/common/tasks/system-file-permissions.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Set permissions for system log directory recurisively
+  file: path=/var/log state=directory recurse=yes mode=0640
```

Comment:
Not in general. You shouldn't typically use acl. What exactly is the line item you are trying to implement?